From 36cd04ff43d5b841908d1380d18838b777e1d928 Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Sat, 1 Dec 2007 00:22:45 +0200
Subject: [PATCH] grepmail: Add CPAN check

Signed-off-by: Jari Aalto <jari.aalto@cante.net>
---
 grepmail |   23 +++++++++++++++++++++--
 1 file changed, 21 insertions(+), 2 deletions(-)

--- a/grepmail
+++ b/grepmail
@@ -31,10 +31,29 @@ use vars qw( %opts $commandLine $VERSION
 use Getopt::Std;
 
 use strict;
-use Mail::Mbox::MessageParser;
-use FileHandle;
 use Carp;
 
+# CPAN MODULES
+
+BEGIN
+{
+    my @list;
+
+    for ( 'Mail::Mbox::MessageParser', 'Mail::Internet' )
+    {
+        eval "use $_;" ;
+        push @list, $_   if $@;  # Eval error
+    }
+
+    if ( @list )
+    {
+        die q([FATAL] Perl CPAN modules are needed. Please install them with:
+
+perl -MCPAN -e 'install $_ for @ARGV' ) . qq(@list\n);
+
+    }
+}
+
 # Set to 1 to enable caching capability
 $USE_CACHING = 1;
 
